elements/rh-accordion/rh-accordion.css

```diff
@@ -5,6 +5,16 @@
   &.expanded {
     box-shadow: var(--rh-box-shadow-sm, 0 2px 4px 0 rgba(21, 21, 21, 0.2));
   }
+
+  &.large {
+    --_header-font-size: var(--rh-font-size-body-text-md, 1rem);
+    --_panel-font-size: var(--rh-font-size-body-text-md, 1rem);
+
+    /* --rh-media-sm */
+    @media (min-width: 768px) {
```

Comment:
I wonder if this should  be a @container query with the component itself the container. Accordions aren't always full-width to the document so the container query would allow accordions placed in a smaller fraction of a page to adjust their size to properly based on their own width rather than the page, but when they are full-width still adjust appropriately.